8/sam-phippen-captions.srt

```diff
@@ -2971,13 +2979,13 @@ Actually, the other thing we could do...
 
 645
 00:58:26,640 --> 00:58:39,360
-is just have a... this... and then we can
-reference bond concom proxies and do things to it
+is just have... this... and then we can
+reference Bond concom proxies and do things to it
```

Comment:
That is a great catch, I had no idea what was going on there, even after a couple of listens. For fidelity, we can then change it to reflect the constant, i.e. `Bond::PROXIES`